rest-api-spec/src/main/resources/rest-api-spec/test/indices.data_stream/40_supported_apis.yml

```diff
@@ -0,0 +1,37 @@
+---
+setup:
+  - do:
+      indices.create_data_stream:
+        name: logs-foobar
+        body:
+          timestamp_field: "@timestamp"
```

Comment:
Pointing out the obvious, but wanting to be sure that we do not merge both this PR and #56888 without removing this field specification (here and in a few other places).

Reply:
Agreed, I will merge #56888 before this pr.